collectd/files/df.conf

```diff
@@ -1,11 +1,21 @@
+{%- from "collectd/map.jinja" import collectd_settings with context %}
+{%- set df_settings = collectd_settings.plugins.df -%}
+
+#
+# DO NOT EDIT
+#
+# This file is managed by salt via {{ source }}
+# Modify the config that generates this file instead
+#
+
 LoadPlugin df
 
 <Plugin df>
-      {{ 'Device "{}"'.format(Device) if Device else '' }}
-      {{ 'MountPoint "{}"'.format(MountPoint) if MountPoint else '' }}
-      {{ 'FSType "{}"'.format(FSType) if FSType else '' }}
-      IgnoreSelected {{ IgnoreSelected }}
-      ReportByDevice {{ ReportByDevice }}
-      ReportReserved {{ ReportReserved }}
-      ReportInodes {{ ReportInodes }}
+      Device "{{ df_settings.Device }}"
+      MountPoint "{{ df_settings.MountPoint }}"
+      FSType "{{ df_settings.FSType }}"
```

Comment:
You got rid of the ability to only specify one of Device, MountPoint, or FSType. You wouldn't generally set all 3.